test/acceptance/integration/notifications/ckan/ckan_column_http.feature

```diff
@@ -0,0 +1,363 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of fiware-connectors (FI-WARE project).
+#
+# fiware-connectors is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+# fiware-connectors is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with fiware-connectors. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact:
+#  iot_support at tid.es
+#
+__author__ = 'Iván Arias León (ivan.ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+Feature: Stored in ckan new notifications per column from context broker
+  As a cygnus user
+  I want to be able to store in ckan new notifications per column from context broker
+  so that they become more functional and useful
+
+  @happy_path
```

Comment:
These multiples tags is most possible its use in Integration Test, because in each scenario are using several features, but in component tests, all scenarios of  a feature are together